src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -75,73 +75,74 @@ void BansheePlaylistModel::setTableModel(int playlistId) {
         m_playlistId = playlistId;
 
         QSqlQuery query(m_database);
-        QString strQuery("CREATE TEMP TABLE IF NOT EXISTS %1"
-            " (" CLM_TRACK_ID " INTEGER, "
-                 CLM_VIEW_ORDER " INTEGER, "
-                 CLM_ARTIST " TEXT, "
-                 CLM_TITLE " TEXT, "
-                 CLM_DURATION " INTEGER, "
-                 CLM_URI " TEXT, "
-                 CLM_ALBUM " TEXT, "
-                 CLM_ALBUM_ARTIST " TEXT, "
-                 CLM_YEAR " INTEGER, "
-                 CLM_RATING " INTEGER, "
-                 CLM_GENRE " TEXT, "
-                 CLM_GROUPING " TEXT, "
-                 CLM_TRACKNUMBER " INTEGER, "
-                 CLM_DATEADDED " INTEGER, "
-                 CLM_BPM " INTEGER, "
-                 CLM_BITRATE " INTEGER, "
-                 CLM_COMMENT " TEXT, "
-                 CLM_PLAYCOUNT" INTEGER, "
-                 CLM_COMPOSER " TEXT, "
-                 CLM_PREVIEW " TEXT)");
-        if (!query.exec(strQuery.arg(m_tempTableName))) {
+        if (!query.exec(QStringLiteral(
+                    "CREATE TEMP TABLE IF NOT EXISTS %1 (" //
```

Comment:
Please remove these empty end-of-line comments that don't belong to the code and don't add any value.

Reply:
I don't know how else to get to a satisfactory result in this code, this is what the auto-formatter does otherwise:
![image](https://user-images.githubusercontent.com/13354331/82602352-3e99b980-9bb1-11ea-8ba2-13339012ab93.png)
This is the only alternative I found, but it is semantically very wrong because the type of one column is in the same line with the name of the next:
![image](https://user-images.githubusercontent.com/13354331/82602472-7143b200-9bb1-11ea-92f6-8d1164471682.png)

Reply:
I got to a satisfactory solution for the part below (https://github.com/mixxxdj/mixxx/pull/2715/commits/bc862eda302bbded3d4790c315db6970d7ec83a4), but this doesn't work here because there's a different type for each column.

Reply:
Are you using pre-commit? If you do you shouldn't get a single long line thanks to the line-length hook.

Reply:
I am using pre-commit now. Still ending up with that long line ^^

Reply:
@daschuer